packages/notte-core/src/notte_core/actions/__init__.py

```
@@ -0,0 +1,149 @@
+    "MultiFactorFillActionDict",
+    "PressKeyActionDict",
+    "ReloadActionDict",
+    "ScrollDownActionDict",
+    "ScrollUpActionDict",
+    "ScrapeActionDict",
+    "SelectDropdownOptionActionDict",
+    "SmsReadActionDict",
+    "SwitchTabActionDict",
+    "UploadFileActionDict",
+    "WaitActionDict",
+]
```

Ruff (0.14.8)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)